src/borg/helpers/parseformat.py

```diff
@@ -963,6 +915,125 @@ def format_iso_time(self, key, item):
         return self.format_time(key, item).isoformat()
 
 
+class DiffFormatter(BaseFormatter):
+    KEY_DESCRIPTIONS = {
+        "path": "archived file path",
+        "change": "all available changes",
+        "content": "file content change",
+        "mode": "file mode change",
+        "type": "file type change",
+        "owner": "file owner (user/group) change",
+        "user": "file user change",
+        "group": "file group change",
```

Comment:
looks like some stuff is missing here (see KEY_GROUPS)?

Reply:
other Formatter also does?

Reply:
ehrm, yeah, looks like. thanks for fixing these.